lms/djangoapps/courseware/views/views.py

```diff
@@ -515,14 +510,23 @@ def course_about(request, course_id):
         ecommerce_bulk_checkout_link = ''
         professional_mode = None
         ecomm_service = EcommerceService()
-        if ecomm_service.is_enabled(request.user) and (
+        _is_ecomm_service_enabled = ecomm_service.is_enabled(request.user)
+        if _is_ecomm_service_enabled and (
                 CourseMode.PROFESSIONAL in modes or CourseMode.NO_ID_PROFESSIONAL_MODE in modes
         ):
             professional_mode = modes.get(CourseMode.PROFESSIONAL, '') or \
                 modes.get(CourseMode.NO_ID_PROFESSIONAL_MODE, '')
             ecommerce_checkout_link = ecomm_service.checkout_page_url(professional_mode.sku)
             if professional_mode.bulk_sku:
                 ecommerce_bulk_checkout_link = ecomm_service.checkout_page_url(professional_mode.bulk_sku)
+
+        registration_price = 0
+        if _is_ecomm_service_enabled or _is_shopping_cart_enabled:
+            registration_price = CourseMode.min_course_price_for_currency(
+                course_key,
+                settings.PAID_COURSE_REGISTRATION_CURRENCY[0]
```

Comment:
Direct index references always bother me because they're brittle -- is there a better way to get the information we need?